sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/config/TransportConfig.java

```diff
@@ -48,12 +52,69 @@ public static Long getHeartbeatIntervalMs() {
     }
 
     /**
-     * Get ip:port of Sentinel Dashboard.
+     * Get a list of (ip/domain, port) indicating Sentinel Dashboard's address.<br>
+     * NOTE: only support <b>HTTP</b> protocol
      *
-     * @return console server ip:port, maybe null if not configured
+     * @return list of (ip/domain, port) pair. <br>
+     *         <b>May not be null</b>. <br>
+     *         An empty list returned when not configured.
      */
-    public static String getConsoleServer() {
-        return SentinelConfig.getConfig(CONSOLE_SERVER);
+    public static List<Tuple2<String, Integer>> getConsoleServerList() {
+        String config = SentinelConfig.getConfig(CONSOLE_SERVER);
+        List<Tuple2<String, Integer>> list = new ArrayList<Tuple2<String, Integer>>();
+        if (StringUtil.isBlank(config)) {
+            RecordLog.warn("Dashboard server address is not configured");
+            return list;
+        }
+        
+        int pos = -1;
+        int cur = 0;
+        while (true) {
+            pos = config.indexOf(',', cur);
+            if (cur < config.length() - 1 && pos < 0) {
+                // for single segment, pos move to the end
+                pos = config.length();
+            }
+            if (pos < 0) {
+                break;
+            }
+            if (pos <= cur) {
+                cur ++;
+                continue;
+            }
+            // parsing
+            String ipPortStr = config.substring(cur, pos);
+            cur = pos + 1;
+            if (StringUtil.isBlank(ipPortStr)) {
+                continue;
+            }
+            ipPortStr = ipPortStr.trim();
+            if (ipPortStr.startsWith("http://")) {
+                ipPortStr = ipPortStr.substring(7);
+            }
+            int index = ipPortStr.indexOf(":");
+            int port = 80;
+            if (index == 0) {
+                // skip
+                continue;
+            }
+            String host = ipPortStr;
+            if (index >= 0) {
+                try {
+                    port = Integer.parseInt(ipPortStr.substring(index + 1));
+                    if (port <= 1 || port >= 65535) {
+                        throw new RuntimeException("Port number [" + port + "] over range");
```

Comment:
Maybe here we could just ignore the invalid item, rather than fail directly. What do you think of it?

Reply:
The dealing in old logics,
it will be just ignored in `simple-http` while in `netty-http` an exception is generated (in `InetSocketAddress.checkPort()`) but also caught immediately in place, log a warning in record.log, print a stack trace to standard output.

Because the logging of sentinel can not be merged into the biz project's logging so the warning may be ignored. So the malformed `:port` is more like to be eaten and initializing flow will go on without the correct dashboard target.
That's why some guys suffer from no heartbeat / no application registered.

I like the idea of stopping to fix rather than ignore but logging. 
But I will also feel ok on keeping the logic the same as how it acted in `netty-http`.

Reply:
It's okay for me. I'll review the PR these days.